stored-procedures/dbo.Cleanup_Msdb.sql

```diff
@@ -45,8 +45,10 @@ EXEC msdb.dbo.sysmail_delete_mailitems_sp @sent_before = @RetainDate;
     
 
 --And do index maintenance
-DECLARE @Sql nvarchar(max) = ''
-SELECT @Sql = @Sql + 'ALTER INDEX [' + i.name + '] ON [msdb].[' + s.name + '].[' + t.name + '] REORGANIZE;' + CHAR(10)
+-- if you aren't doing index maintenance on msdb, we'll force it here
+-- sometimes this prevents significant space bloat, esp on older versions
+DECLARE @Sql nvarchar(max) = N''
+SELECT @Sql = @Sql + N'ALTER INDEX ' + QUOTENAME(i.name) + N' ON [msdb].' + QUOTENAME(s.name) + N'.' + QUOTENAME(t.name) + N' REORGANIZE;' + CHAR(10)

```

Comment:
And if you're in here I'd change sp_executesql to sys.sp_executesql

Reply:
Good catch! 👍